src/commands/test/lib/included-files-parser.ts

```diff
@@ -1,17 +1,68 @@
 import { IFileDescriptionJson } from "./test-manifest-reader";
+import * as path from "path";
 
-export function parseIncludedFiles(includedFiles: string[]): IFileDescriptionJson[] {
-  return includedFiles.map(f => parseIncludedFile(f));
+const invalidCharactersRegexp = /['"!#$%&+^<=>`|]/
+
+export function parseIncludedFiles(includedFiles: string[], rootDir: string): IFileDescriptionJson[] {
+  return includedFiles.map(f => parseIncludedFile(f, rootDir));
 }
 
-function parseIncludedFile(includedFile: string): IFileDescriptionJson {
+function parseIncludedFile(includedFile: string, rootDir: string): IFileDescriptionJson {
   let separatorIndex = includedFile.indexOf("=");
   if (separatorIndex === -1) {
-    throw new Error(`Invalid definition of included files: ${includedFile}. Included files must specified in format "targetPath"="sourcePath"`);
+    return parseIncludedFileFromSinglePath(includedFile, rootDir);
+  }
+  else {
+    return parseIncludedFileFromPathsPair(includedFile, rootDir, separatorIndex);
+  }
+}
+
+function parseIncludedFileFromSinglePath(includedFile: string, rootDir: string): IFileDescriptionJson {
+  validatePath(includedFile);
+
+  if (path.isAbsolute(includedFile)) {
+    let targetPath = path.relative(rootDir, includedFile);
+    if (targetPath.indexOf("..") != -1) {
+      throw new Error(`Invalid included file: "${includedFile}". ` + 
+                      `If only a single path is used, it must be inside directory "${rootDir}"`);
+    }
+
+    return {
+      targetPath: path.relative(rootDir, includedFile),
+      sourcePath: includedFile
+    }
+  }
+  else {
+    return {
+      targetPath: includedFile,
+      sourcePath: path.join(rootDir, includedFile)
+    }
+  }
+}
+
+function parseIncludedFileFromPathsPair(includedFile: string, rootDir: string, separatorIndex: number): IFileDescriptionJson {
+  let targetPath = includedFile.substr(0, separatorIndex);
```

Comment:
Most of the time, using a package for abstracting these path manipulations can be a good idea. For NodeJS, there's https://nodejs.org/docs/latest/api/path.html. Just a suggestion.